Gemfile.lock

```diff
@@ -24,18 +31,29 @@ GEM
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.5.0)
     rspec-support (3.5.0)
+    rubocop (0.58.2)
```

Comment:
if you are adding rubocop... can you split this off into a separate group in bundler so it doesn't get mixed up with real dependencies